keyboards/seigaiha/info.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Needs to be redone without the rotation.
![image](https://user-images.githubusercontent.com/4781841/117996610-a24e6100-b385-11eb-9dda-c66c4aafbe18.png)

Reply:
gotcha! on it:)

Reply:
Done @fauxpark !